### PR TITLE
Add some commands and events in hci

### PR DESCRIPTION
**Pull request by benquike**

### CODE REVIEW

File: bumble/hci.py [hunk 1]
```diff
@@ -2321,6 +2370,160 @@ class HCI_Enhanced_Setup_Synchronous_Connection_Command(HCI_Command):
...
+        ('afh_channel_map', 10),
+    ],
+    return_parameters_fields=[
+        ('status', STATUS_SPEC),
+        ('bd_addr', Address.parse_address),
+    ],
+)
+class HCI_Set_Connectionless_Peripheral_Broadcast_Receive_Command(HCI_Command):
+    '''
+    See Bluetooth spec @ Set Connectionless Peripheral Broadcast Receive Command
```
Comment (barbibulle) [inline bumble/hci.py:2481]: nit: `@ 7.1.50`

File: bumble/hci.py [hunk 2]
```diff
@@ -2321,6 +2370,160 @@ class HCI_Enhanced_Setup_Synchronous_Connection_Command(HCI_Command):
...
+        ('supervision_timeout', 2),
+        ('remote_timing_accuracy', 1),
+        ('skip', 1),
+        ('packet_type', 2),
+        ('afh_channel_map', 10),
+    ],
+    return_parameters_fields=[
+        ('status', STATUS_SPEC),
+        ('bd_addr', Address.parse_address),
+    ],
```
Comment (barbibulle) [inline bumble/hci.py:2477]: I think this is missing `lt_addr`

File: bumble/hci.py [hunk 3]
```diff
@@ -2321,6 +2370,160 @@ class HCI_Enhanced_Setup_Synchronous_Connection_Command(HCI_Command):
...
+    '''
+    See Bluetooth spec @ 7.1.51 Start Synchronization Train Command
+    '''
+
+
+# -----------------------------------------------------------------------------
+@HCI_Command.command(
+    fields=[
+        ('bd_addr', Address.parse_address),
+        ('sync_timeout', 2),
```
Comment (barbibulle) [inline bumble/hci.py:2496]: In the spec, this is called `sync_scan_timeout`

**Review by barbibulle — APPROVED** [review]

File: bumble/hci.py (hunk 1 above)
Comment (benquike) [reply bumble/hci.py]: Updated

File: bumble/hci.py (hunk 3 above)
Comment (benquike) [reply bumble/hci.py]: updated

File: bumble/hci.py (hunk 2 above)
Comment (benquike) [reply bumble/hci.py:2478]: udpated

**Pull request merged**